cmd/autoheal/metrics_exporter.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	promHandler = promhttp.Handler()
+
+	healsInitiated = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "autoheal_heals_initiated_total",
+			Help: "Number of initiated heals",
+		},
+		[]string{"heal_type", "template_name", "heal_name"},
+	)
+)
+
+func initExportedMetrics() {
+	prometheus.MustRegister(healsInitiated)
+}
+
+func healInitiated(labels map[string]string) {
```

Comment:
Can we pass here a pointer to the action object instead of a map? That way the names of the labels only need to be part of this file. Would be nice if we can also avoid repeating them, using constants, for example.

Reply:
Done, I had to pass also the rule name